portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Memo!</h1>");
+    //response.getWriter().println("<h1>Hello Memo!</h1>");
```

Comment:
Remove the commented line. (It's useful to do them during development, but remove them before you submit.)

Reply:
Got it, I'll make sure to remove it